ports/mimxrt/machine_rtc.c

```diff
@@ -25,15 +25,126 @@
  * THE SOFTWARE.
  */
 
+#include "py/mperrno.h"
 #include "py/runtime.h"
+#include "shared/runtime/mpirq.h"
 #include "shared/timeutils/timeutils.h"
 #include "modmachine.h"
 #include "ticks.h"
 #include "fsl_snvs_lp.h"
 
+STATIC mp_int_t timeout = 0;
+
+void machine_rtc_alarm_clear_en() {
+    SNVS_LP_SRTC_DisableInterrupts(SNVS, SNVS_LPCR_LPTA_EN_MASK);
+    while (SNVS->LPCR & SNVS_LPCR_LPTA_EN_MASK) {
+        ;
```

Comment:
the coding style for empty loops is to just have them empty, ie remove this line with a semicolon